packages/contracts/contracts/core/IDAO.sol

```diff
@@ -74,4 +74,25 @@ abstract contract IDAO {
     ) external virtual;
 
     event Withdrawn(address indexed token, address indexed to, uint256 amount, string _reference);
+
+    /// @notice Setter for the trusted forwarder verifying the meta transaction
+    /// @param _trustedForwarder the trusted forwarder address
+    /// @dev used to update the trusted forwarder
+    function setTrustedForwarder(address _trustedForwarder) external virtual;
```

Comment:
i am not sure if i fully understand what is the rational behind adding manually interface for inherited contracts inside DAO.sol?

when an interface is needed in another contract why don't use something like:
`IBaseRelayRecipient(dao).setTrustedForwarder()` ??

Reply:
@novaknole  convinced me to not inherit from `BaseRelayRecipient` or `IRelayRecipient` because the `DAO` shouldn't send Meta transactions. Accordingly, it doesn't need the `_msgData` and `_msgSender` context that comes with `BaseRelayRecipient` or `IRelayRecipient`.
Thus, the decision was to just give it `setTrustedForwarder(address)` and `trustedForwarder() -> address`.
We can also revisit this design question. I just did this change to unblock `MerkleDistributor` and `MerkleMinter` becoming `Component`s. The old way (`trustedForwarder` was not part of `IDAO`) blocked passing on the `trustedForwarder` of `DAO`.

Reply:
my example was in general
for example why not use:
ERC1271(dao).isValidSignature(.....)
?

Reply:
explaining and the sync meeting...

Reply:
I opened [APP-431](https://aragonassociation.atlassian.net/browse/APP-431) to resolve this discussion later.